data/human/human missions.txt

```diff
@@ -6301,3 +6301,479 @@ mission "Blind Man from Martini"
+				decline
+
+mission "Southern Fiance 5"
+	minor
+	landing
+	source "Cornucopia"
+	to offer
+		has "event: carl elected governor"
+		not "event: tom and carl's child"
+	on offer
+		"visited tom and carl" ++
+		conversation
+			branch already
+				has "visited tom and carl"
+			`When you're making your approach for landing, doing your normal back and forth with the controller at the spaceport, you're surprised to hear them ask you to divert to a special pad. You do so, and when you land, you find Tom, the man you brought south from the Core, and his husband Carl, the new Governor of Cornucopia, waiting for you.`
+			`	They greet you like an old friend (though you didn't speak two words to Carl when you were here), and insist on serving you dinner at their house. The Cornucopia governor's mansion is an impressive sight, and Tom takes you on a tour while Carl fields some important phone calls. When you sit down to dinner, you're blown away: it's probably the best food you've had, which shouldn't be a surprise from a planet that's almost entirely covered in farms. You swap stories for a while - you telling about your "thrilling adventures in space", as Tom describes them, and them telling both about some of their ordinary times here on Cornucopia, and about some of the challenges and highlights of the campaign for Governor. They also hint about something big still to come, sharing a knowing look but saying no more.`
+				goto end
+			
+			label already
+			`During your approach, you're not surprised, this time, to see that you've been diverted to the Governor's own landing pad. Tom and Carl are waiting for you once more, and they greet you warmly. They take you in an official government vehicle to the governor's mansion, with a trying-to-be-subtle security detail always following just on the periphery.`
+			`	The official mansion is much more impressive than Carl's house as Agricultural Minister was, and Tom gives you a tour while Carl has to field some official phone calls. You're already looking forward to the food well before you sit down to dinner, and you are not disappointed: if anything, it's better than the last time you visited them. Carl confides that he's planning on pushing to open a government-sponsored culinary school in the capital, but he'll be sad to see his cook go: he knows it's been the woman's dream for ages to teach others to make food as amazing as this, too.`
+			`	After dinner, as before, you catch each other up on what you've been doing since your last visit, and once you've both shared most of what you can think of to tell, Tom and Carl once again hint that there's still something big in their plans, but will not elaborate.`
```

Comment:
```suggestion
			`	After dinner, as before, you catch each other up on what you've been doing since your last visit, and once you've both shared most of what you can think of to tell, Tom and Carl once again hint that there's still something big in their plans.`
```